components/mentorDashboard/calendar.js

```diff
@@ -1,22 +1,23 @@
 import React, { useState } from "react";
 import styles from "../../styles/dashboard.module.css";
```

Comment:
actually yeh bdia h ... schedule m 1. schedule 2. Calender
and routing kch aisi kr lenge `tab=schedule&subTab=scedule` or `tab=schedule&subTab=calender ` how it is?

Reply:
> actually yeh bdia h ... schedule m 1. schedule 2. Calender and routing kch aisi kr lenge `tab=schedule&subTab=scedule` or `tab=schedule&subTab=calender ` how it is?

Isn't is same?